src/api.v2/helpers/worker/generateEtag.js

```diff
@@ -6,36 +6,19 @@ const getExtraDependencies = require('./workSubmit/getExtraDependencies');
 const createObjectHash = (object) => hash.MD5(object);
 
 
-// Disable unique keys to reallow reuse of work results in development
-const DISABLE_UNIQUE_KEYS = [
-  'GetEmbedding',
-];
-
-// we want to disable cache in staging & development
-// however trajectory & download rds depend on the embedding's ETag so we have to
-// disable the unique keys for the embeddings task
-const getCacheUniquenessKey = (taskName) => {
-  // allow people to enable cache in development by setting USE_CACHE=true
-  if (process.env.USE_CACHE === 'true') {
-    return null;
-  }
-
-  if (config.clusterEnv !== 'production' && !DISABLE_UNIQUE_KEYS.includes(taskName)) {
-    return Math.random();
-  }
-  return null;
-};
-
 const generateETag = async (
   data,
 ) => {
   const {
     experimentId,
     body,
-    extras,
```

Comment:
I'm not sure this removal of extras is fine as it is done right now.

In the UI I saw at least one clear instance where an `extras` is being used (`getDiffExprGenes`). There might be other instances too

Reply:
do you mean the pagination? or which part of `getDiffExprGenes` ?